StripeCore/StripeCore/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this be added to the StripeCore podspec?

Reply:
Oh weird, it's open in my text editor but there was an error when saving the .rb file. Committing it now